src/ResourceManager/Websites/Commands.Websites/Cmdlets/BackupRestore/RestoreAzureWebAppSnapshot.cs

```diff
@@ -48,23 +48,23 @@ public class RestoreAzureWebAppSnapshot : WebAppOptionalSlotBaseCmdlet
         public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
-            Site targetApp = WebsitesClient.GetWebApp(ResourceGroupName, Name, Slot);
-            SnapshotRecoveryTarget target = new SnapshotRecoveryTarget()
+            Site sourceApp = WebsitesClient.GetWebApp(InputObject.ResourceGroupName, InputObject.Name, InputObject.Slot);
+            SnapshotRecoverySource source = new SnapshotRecoverySource()
             {
-                Location = targetApp.Location,
-                Id = targetApp.Id
+                Location = sourceApp.Location,
+                Id = sourceApp.Id
             };
-            SnapshotRecoveryRequest recoveryReq = new SnapshotRecoveryRequest()
+            SnapshotRestoreRequest recoveryReq = new SnapshotRestoreRequest()
             {
                 Overwrite = true,
                 SnapshotTime = this.InputObject.SnapshotTime.ToString("o"),
                 RecoverConfiguration = this.RecoverConfiguration,
                 IgnoreConflictingHostNames = true,
-                RecoveryTarget = target
+                RecoverySource = source
             };
-            Action recoverAction = () => WebsitesClient.RecoverSite(InputObject.ResourceGroupName, InputObject.Name, InputObject.Slot, recoveryReq);
+            Action recoverAction = () => WebsitesClient.RestoreSnapshot(ResourceGroupName, Name, Slot, recoveryReq);
```

Comment:
Same on these two lines, was this a bug before?

Reply:
Same, see above